README.md

```diff
@@ -1,4 +1,6 @@
 [![Build Status](https://travis-ci.org/firebase/firebase-admin-python.svg?branch=master)](https://travis-ci.org/firebase/firebase-admin-python)
+[![Python](https://img.shields.io/pypi/pyversions/firebase-admin.svg)](https://pypi.org/project/firebase-admin/)
```

Comment:
This svg still shows 3.4, I assume it will be updated later?

Reply:
Yes, it shows the info for the last release. Will automatically update once we do another release.